orchestration/roles/app/tasks/setup_virtualenv.yml

```diff
@@ -3,17 +3,37 @@
 - name: Ensure virtualenvwrapper is installed
   pip: name=virtualenvwrapper executable="/usr/bin/pip3"
 
-- name: Export WORKON_HOME in .bashrc
-  lineinfile: dest=/home/{{ gunicorn_user }}/.bashrc line="export WORKON_HOME=$HOME/.virtualenvs"
+- name: Export WORKON_HOME in /root/.bashrc
+  lineinfile: dest=/root/.bashrc line="export WORKON_HOME=/server/.virtualenvs"
+  when: env != "vagrant"
 
-- name: Export PROJECT_HOME in .bashrc
-  lineinfile: dest=/home/{{ gunicorn_user }}/.bashrc line="export PROJECT_HOME=$HOME/Devel"
+- name: Export PROJECT_HOME in /root/.bashrc
+  lineinfile: dest=/root/.bashrc line="export PROJECT_HOME=/server/Devel"
```

Comment:
What's the `Devel` folder for? might be more explicit to call this folder `project` or `app`. Also probably want to keep it all lowercase.